cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -289,11 +289,7 @@ func (cm *certManagerClient) migrateCRDs() error {
 		return err
 	}
 
-	if err := newCRDMigrator(c).Run(ctx, objs); err != nil {
-		return err
```

Comment:
if someone wanted to wrap the error and add more context to it, will the linter complain?

Reply:
I think that would be fine. I just removed the err assignment since it wasn't needed as the implementation is right now.